docs/src/test/scala/docs/http/scaladsl/server/directives/SecurityDirectivesExamplesSpec.scala

```diff
@@ -10,144 +10,176 @@ import akka.http.scaladsl.server.Route
 import akka.http.scaladsl.server.RoutingSpec
 import akka.http.scaladsl.server.directives.Credentials
 import docs.CompileOnlySpec
+import org.scalatest.prop.TableDrivenPropertyChecks._
 
 import scala.concurrent.Future
 
 class SecurityDirectivesExamplesSpec extends RoutingSpec with CompileOnlySpec {
 
+  def verifier(secret: String)(receivedSecret: String): Boolean = secret == receivedSecret
+
   "authenticateBasic-0" in {
     //#authenticateBasic-0
-    def myUserPassAuthenticator(credentials: Credentials): Option[String] =
+    def myUserPassAuthenticatorVerify(credentials: Credentials): Option[String] =
```

Comment:
This is pulled into documentation, I'd expect we want to keep it unchanged and only add new test cases/examples for the new `provideVerify`?

Reply:
Yes, I am not sure where should I add new test for this, do we need to update the documentation test or is there any other file that test those SecurityDirectives?

Reply:
I think I found it, there wasn't any test for verify function I added one. Let me know if its all good.